compose/dumpsql.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-TAG=18.09
+TAG=v18.09
```

Comment:
should that be 18.09?

Reply:
I tested, but no, it failed (=empty dump) without the `v`, because all images [are tagged with this v](https://github.com/bgruening/docker-galaxy-stable/blob/dev/compose/build-orchestration-images.sh#L140)

Reply:
mh, ok I might remove this at some point. We should standardize it ...